.gitignore

```diff
@@ -87,4 +87,8 @@ ignite/
 *.classpath
 *.project
 *.prefs
-*.factorypath
\ No newline at end of file
+*.factorypath
+
```

Comment:
@kiran1048 can you add/update instruction on how to run the script so that other members in the community could leverage this script? Thanks.